src/server/server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const morgan = require('morgan');
+const path = require('path');
+
+const app = express();
+
+app.use(morgan('dev'));
+app.use('/', express.static(path.join(__dirname, '../client')));
```

Comment:
nothing huge again, but might be a good idea to just test the actual route youll be using instead of '/', just a thought